src/css/base/_grid.scss

```diff
@@ -1,36 +1,40 @@
-.wrap {
+.wrap, div.wrap {
     @include wrap;
 }
-.grid {
-    display: flex;
-    align-items: stretch;
-    flex-wrap: wrap;
-    margin-left: calc(var(--gutter)*-.5);
-    margin-right: calc(var(--gutter)*-.5);
-}
-.row {
-    display: flex;
-    align-items: stretch;
-    margin-left: calc(var(--gutter)*-.5);
-    margin-right: calc(var(--gutter)*-.5);
+
+
+//.grid should be the default grid container, but previous classnames have
+// been included to ease the transition from flexbox
+.grid, .row, .wrap:has(> .col) {

```

Comment:
I think it would be fine to only support .grid.

I don't think we need to worry about backwards compatibility, supporting more ways to do this might get confusing, and .row might be better reserved for flex stuff?